app/assets/javascripts/bootstrap.js.coffee

```diff
@@ -1,4 +1,4 @@
 jQuery ->
   $("a[rel~=popover], .has-popover").popover()
   $("a[rel~=tooltip], .has-tooltip").tooltip()
-  $('.datepicker').datepicker()
\ No newline at end of file
+  $('.datepicker').datepicker()
```

Comment:
Ich hätte gedacht der Linebreak wurde hinzugefügt :) "No newline at end of file"